test/src/test/java/org/corfudb/recovery/Helpers.java

```diff
@@ -0,0 +1,107 @@
+package org.corfudb.recovery;
+
+import org.corfudb.protocols.wireprotocol.DataType;
+import org.corfudb.protocols.wireprotocol.ILogData;
+import org.corfudb.protocols.wireprotocol.IMetadata;
+import org.corfudb.protocols.wireprotocol.LogData;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.collections.CorfuTable;
+import org.corfudb.runtime.collections.SMRMap;
+import org.corfudb.runtime.object.CorfuCompileProxy;
+import org.corfudb.runtime.object.ICorfuSMR;
+import org.corfudb.runtime.object.VersionLockedObject;
+import org.corfudb.runtime.view.ObjectsView;
+
+import java.util.Map;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Created by rmichoud on 10/23/17.
+ */
+public class Helpers{
+    static ILogData.SerializationHandle createEmptyData(long position, DataType type, IMetadata.DataRank rank) {
+        ILogData data = new LogData(type);
+        data.setRank(rank);
+        data.setGlobalAddress(position);
+        return data.getSerializedForm();
+    }
+
+    static Map<String, String> createMap(String streamName, CorfuRuntime cr) {
+        return createMap(streamName, cr, SMRMap.class);
+    }
+
+    static <T> Map<String, String> createMap(String streamName, CorfuRuntime cr, Class<T> type) {
+        return (Map<String, String>) cr.getObjectsView().build()
+                .setStreamName(streamName)
+                .setType(type)
+                .open();
+    }
+
+    static CorfuCompileProxy getCorfuCompileProxy(CorfuRuntime cr, String streamName, Class type) {
+        ObjectsView.ObjectID mapId = new ObjectsView.
+                ObjectID(CorfuRuntime.getStreamID(streamName), type);
+
+        return ((CorfuCompileProxy) ((ICorfuSMR) cr.getObjectsView().
+                getObjectCache().
+                get(mapId)).
+                getCorfuSMRProxy());
+    }
+
+    static VersionLockedObject getVersionLockedObject(CorfuRuntime cr, String streamName, Class type) {
+        CorfuCompileProxy cp = getCorfuCompileProxy(cr, streamName, type);
+        return cp.getUnderlyingObject();
+    }
+
+    static void assertThatMapIsNotBuilt(CorfuRuntime rt, String streamName, Class type) {
+        ObjectsView.ObjectID mapId = new ObjectsView.
+                ObjectID(CorfuRuntime.getStreamID(streamName), type);
+
+        assertThat(!rt.getObjectsView().getObjectCache().containsKey(mapId));
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Complete the assertion. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2970)